<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(92, 131, 116);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_numeros">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Números:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_suma">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resultado:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_numeros">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>81</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(236, 236, 236);
border: 5px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_resultado">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(236, 236, 236);
border: 5px solid black;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_moda">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 200, 185);
border: 5px solid black;
border-radius: 20px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>MODA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mediana">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 200, 185);
border: 5px solid black;
border-radius: 20px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>MEDIANA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_promedio">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 200, 185);
border: 5px solid black;
border-radius: 20px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>PROMEDIO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_desvestandar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 200, 185);
border: 5px solid black;
border-radius: 20px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>DESV. ESTÁNDAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_valormayor">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 200, 185);
border: 5px solid black;
border-radius: 20px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>VALOR MAYOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_valormenor">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 200, 185);
border: 5px solid black;
border-radius: 20px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>VALOR MENOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_varianza">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 200, 185);
border: 5px solid black;
border-radius: 20px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>VARIANZA</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
